src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -44,13 +55,71 @@ public void playGame(){
                 promptNextPlayer();
             }
         }
-
         board.printBoard();
-        System.out.println("Player " + board.getWinner() + " has won the game!");
+
+        updateScoreboard(board.getWinner());
+
+        if (board.getWinner() == Player.NONE) {
+            System.out.println("This is a tie!");
+        }
+        else {
+            System.out.println("Player " + board.getWinner() + " has won the game!");
+        }
+
+        printScoreboard();
+    }
+
+    private void updateScoreboard(Player winner) {
+        if (winner == Player.NONE) {
+            scoreboard[0][2] += 1;
```

Comment:
[Desired] ```scoreboard[0][2]``` is somewhat ambiguous and it is not immediately clear what values are being tallied. Consider adding constant values for scoreboard win/loss types to improve readability and clarity within the code. 

```suggestion
            scoreboard[0][TIE] += 1;
```

Reply:
Fixed by adding more comments.

Reply:
Great, thanks.